crescent/commands/signature.py

```diff
@@ -45,7 +45,7 @@ def _unwrap_optional(origin: type[Any]) -> Any:
     return args[1]
 
 
-def _get_arg(t: type[Arg] | type[Any], metadata: Iterable[Any]) -> T | None:
+def _get_arg(t: type[Arg] | type[Any], metadata: Iterable[Any]) -> T | None:  # pyright: ignore
```

Comment:
What's being ignored here? Maybe add a comment, and also put each parameter on it's own line and ignore just one of those.

Reply:
It's the typevar invalid use thing. In this case, would it make sense to leave it as is? I can leave additional info.

Reply:
wait... why is there a typevar

Reply:
Any updates on this ^

Reply:
@Lunarmagpie why is there a typevar here and what does it reference?

Reply:
Look at the next line. I think this can be `Any` though.

Reply:
TypeVars don't work like that 😩


<img width="271" alt="Screen Shot 2022-06-22 at 4 50 15 PM" src="https://user-images.githubusercontent.com/55721663/175133589-3fac8c19-a051-494f-8927-dda784a2a79c.png">
<img width="728" alt="Screen Shot 2022-06-22 at 4 50 26 PM" src="https://user-images.githubusercontent.com/55721663/175133614-2013215c-9180-444e-9cb4-509f2da0bd9c.png">

Reply:
so true. I question why tf i wrote that.

Reply:
Probably just make it Any then?